Pull request for adding mobile menu in my portfolio

Hamayounali opened the pull request:
1. Updated style.css file. 
2. Updated index.html file. 
3. Created script.js file

Hamayounali committed:
Create mobile menu

1. Updated style.css file
2. Updated index.html file
3. Created script.js file

Hamayounali committed:
Merge branch 'working-branch' of https://github.com/Hamayounali/hamayoun-portfolio into working-branch

Hamayounali committed:
Updated Mobile menu

Hamayounali committed:
Update style.css

Hamayounali committed:
Update style.css

Hamayounali committed:
Update style.css

Hamayounali committed:
Update style.css

Hamayounali committed:
Update style.css

vechicin commented on assets/css/style.css:
```diff
@@ -878,3 +729,84 @@ body {
 }
+
+/* Memu barr */
+.topnav {
+  overflow: hidden;
+  background-color: #333;
+  padding-left: 90px;
+  padding-right: 90px;
+}
```
Your design doesn't quite look like the template design.

Your design:

<img width="144" alt="Screen Shot 2022-06-20 at 10 23 39" src="https://user-images.githubusercontent.com/82426682/174634329-923c272b-9fc5-43d8-8451-424c6ab497f5.png">

The template design:

<img width="191" alt="Screen Shot 2022-06-20 at 10 24 15" src="https://user-images.githubusercontent.com/82426682/174634418-cfa5e831-2bc3-4666-a212-30ab831cc363.png">

Please fix it.

vechicin commented on assets/css/style.css:
```diff
@@ -878,3 +729,84 @@ body {
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .topnav.responsive {
+    position: relative;
+  }
+
+  .topnav.responsive .icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
+}
```
According to the principles of mobile-first development, media queries should only be used to make the design responsive for bigger screens. This media query is being applied for mobile screens. Please remove it.

Additionally, your design is not following the template design.

Your design:

<img width="145" alt="Screen Shot 2022-06-20 at 10 26 44" src="https://user-images.githubusercontent.com/82426682/174634887-0aebbc85-e587-4264-b2fe-408e06f8a2bc.png">

Template design:

<img width="191" alt="Screen Shot 2022-06-20 at 10 27 12" src="https://user-images.githubusercontent.com/82426682/174634982-fdcdeb91-11ce-4e1d-9096-0c67c02f3ff6.png">

Please fix it.

Additionally, the mobile menu behavior is not the one expected. Please review what happens when the user clicks on the mobile menu links. Please fix the behavior.

sja-thedude replied:
- [x] As suggested previously kindly remove media queries from the mobile version as it is only needed for the desktop.
- [x] Kindly make sure to follow the same mobile menu design as per the Figma design. Follow the same colour, spacing and font-weight for the same. Also, make sure to display the line after each menu option as per the design.

https://user-images.githubusercontent.com/70269507/174764938-04cd3c44-3e80-431a-9176-2264dfbe4e24.mov

vechicin reviewed (changes requested):
# Hi @Hamayounali ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good first attempt at implementing the mobile menu

## Required Changes ♻️

- [ ] You need to install ESLint since this project includes JavaScript. Please do so.

<img width="775" alt="Screen Shot 2022-06-20 at 10 31 13" src="https://user-images.githubusercontent.com/82426682/174635667-dff26520-5c01-4945-8855-522b39005076.png">

- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) there needs to be contributions by both partners. I can only see one partner's contribution. If you are having issues with your coding partner, please reach out to Student Success and make a note in your review request about it.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hamayounali committed:
installed eslint

Hamayounali committed:
Update style.css

Hamayounali committed:
Update script.js

Hamayounali committed:
Update style.css

MxoDevops committed:
added eslint file & .gitignore

MxoDevops committed:
deleted node_modules

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @Hamayounali, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![200w](https://user-images.githubusercontent.com/70269507/159484303-3187dacc-4a82-4263-bb1a-8d72417ce1d9.gif)

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hamayounali committed:
Updated style.css

Hamayounali committed:
Update style.css

Hamayounali commented:
Thank you Sayeda for the nice comments and review. I have removed the media query and updated the menu accordingly. I hope this time my project meets the requirements.

sja-thedude commented on assets/css/style.css:
```diff
@@ -877,4 +728,92 @@ body {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
```
- [x] As suggested previously, kindly make changes as per the Figma design.
- [x] Your design has to be similar to the design on the left as per Figma.
- [x] The mobile menu should be of full length not half
- [x] The envelope icon should not display on the mobile menu and you should follow the same color text for menu options.

![2022-06-21](https://user-images.githubusercontent.com/70269507/174793639-537867bf-c044-4b9a-a71c-f8edc43a0b26.png)

Hamayounali replied:
Thank you @sja-thedude for your time and nice review. I have updated the menu according to your comments.

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @Hamayounali, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![200w](https://user-images.githubusercontent.com/70269507/159484303-3187dacc-4a82-4263-bb1a-8d72417ce1d9.gif)

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hamayounali committed:
updated style.css

Hamayounali committed:
Update style.css

rukundoeric commented on index.html:
```diff
@@ -5,6 +5,8 @@
 	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="assets/js/script.js"></script>
 	<style>
 
 		@font-face {
```
- [ ] <b>OPTIONAL:</b> It is not best practice to use internal CSS. It would nice to consider moving them to external file.

rukundoeric reviewed (approved):
Hi @Hamayounali,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] check the comments under review

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Hamayounali committed:
Portfolio details popup window

1. Updated index.html file
2. Updated style.css file
3. Updated script.js file
4. Created popup.js file

Hamayounali committed:
Revert "Portfolio details popup window"

This reverts commit c8dd9244fc747dded6df89bc8b68e9bfdfd56641.

Hamayounali committed:
Revert "Revert "Portfolio details popup window""

This reverts commit dd19660238c1de37636e282968c58bd8e36f67c0.

Hamayounali committed:
Revert "Portfolio details popup window"

This reverts commit c8dd9244fc747dded6df89bc8b68e9bfdfd56641.
